bench/worker/worker_ci_bench.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'base'
+
+class WorkerCIBenchmark < Base
```

Comment:
What is the purpose of splitting the two?
They seem identical, just that the non-CI also runs with 1M reports?

I think we can either parameterize it, or enable/disable 1M for both modes.

Or, if you want to keep both, maybe refactor the `run` method to reuse most of the code.

Reply:
Done in https://github.com/3scale/apisonator/pull/403/commits/63b5c2d17b32271c7f638abc5d80736ce2baa561

Reply:
To me this CI class also looks not very useful. Maybe use an env-variable for the number. But whatever.

Reply:
> To me this CI class also looks not very useful.

Have you checked https://github.com/3scale/apisonator/commit/63b5c2d17b32271c7f638abc5d80736ce2baa561 ? `WorkerBenchmark` and `WorkerCIBenchmark` do different things, one runs two benchmarks and the other one runs one benchmark + gets memory usage. Any ideas?

> Maybe use an env-variable for the number. But whatever.

For which number exactly? I understand this as reusing `WorkerBenchmark` but getting the amount of reports from an env var. But `WorkerBenchmark` runs two benchmarks with two amounts.

Reply:
I don't think we need to stick to this. I'm fine running just ones either way.